Reference to selector widgets.mdx (#9808)
getsentry/sentry-docs: docs/product/session-replay/replay-page-and-filters.mdx

- **Dead Clicks**: User clicks on `a` and `button` tags that do not result in any page activity after 7 seconds (i.e. no HTML was added, removed, or updated; no visual changes were observed in the page).
- **Rage Clicks**: Five or more clicks on a dead element (it exhibits no page activity after 7 seconds.) Rage clicks are a subset of dead clicks.

> The **Replay** page also shows two widgets titled "Most Dead Clicks" and "Most Rage Clicks" that show the top selectors that have been rage or dead clicked on. Expanding the selector will show example replays that contain clicks on that selector. You can also click the "See all selectors" button to see all selectors that have been rage or dead clicked on. SDK version `7.60.1` or higher is required to see rage and dead click data on the **Replay** page.

Author: ```suggestion
The **Replay** page also shows two widgets titled "Most Dead Clicks" and "Most Rage Clicks" that displays the top selectors that have been rage or dead clicked on. Expanding the selector will show example replays that contain clicks on that selector. You can also click the "See all selectors" button to see all selectors that have been rage or dead clicked on. SDK version `7.60.1` or higher is required to see rage and dead click data on the **Replay** page.
```

> The **Replay** page also shows two widgets titled "Most Dead Clicks" and "Most Rage Clicks" that show the top selectors that have been rage or dead clicked on. Expanding the selector will show example replays that contain clicks on that selector. You can also click the "See all selectors" button to see all selectors that have been rage or dead clicked on. SDK version `7.60.1` or higher is required to see rage and dead click data on the **Replay** page.

Reviewer: ```suggestion
The **Replay** page also has two widgets titled "Most Dead Clicks" and "Most Rage Clicks" that show the selectors with the most rage or dead clicks. Expanding the selector will show example replays where that selector was clicked. You can also click the "See all selectors" button to view all selectors that have gotten rage or dead clicks. SDK version `7.60.1` or higher is required to see rage and dead click data on the **Replay** page.
```
